Sparkle/SUAppcast.m

```diff
@@ -128,7 +128,7 @@ - (NSDictionary *)attributesOfNode:(NSXMLElement *)node
         if (!attrName) {
             continue;
         }
-        [dictionary setObject:[attribute stringValue] forKey:attrName];
+        [dictionary setValue:[attribute stringValue] forKey:attrName];
     }
```

Comment:
> Alternatively, we could assign stringValue to a local variable and test it for nil. I just tested that, and clang is happy that way too. What do you think?

Yeah, this is what I meant, but I was in a rush. Thank you for the pull request.